[CTOR-1163]-Refresh hardware-devices-timelinkmicro-tms-6001-snmp doc … (#3943)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/hardware-devices-timelinkmicro-tms6001-snmp.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> ## Overview
> 

Reviewer: ```suggestion
```

> TimeLink microsystems provides solutions based on a complete line of COTS products and
> design specific equipment or system on user's requirements. 

Reviewer: ```suggestion
```

## Connector dependencies

The following monitoring connectors will be installed when you install the **Timelinkmicro Tms6001** connector through the
**Configuration > Monitoring Connector Manager** menu:
* [Base Pack](./base-generic.md)

## Pack assets

### Templates

The Monitoring Connector **Timelinkmicro Tms6001** brings a host template:

* **HW-Device-Timelinkmicro-Tms6001-SNMP-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="HW-Device-Timelinkmicro-Tms6001-SNMP-custom" label="HW-Device-Timelinkmicro-Tms6001-SNMP-custom">

| Service Alias | Service Template                                       | Service Description              |
|:--------------|:-------------------------------------------------------|:---------------------------------|

> | Alarms        | HW-Device-Timelinkmicro-Tms6001-Alarms-SNMP-custom     | Check alarms NTP serveur         |

Reviewer: ```suggestion
| Alarms        | HW-Device-Timelinkmicro-Tms6001-Alarms-SNMP-custom     | Check alarms for the NTP server         |
```

| Antenna       | HW-Device-Timelinkmicro-Tms6001-Antenna-SNMP-custom    | Check antenna connector          |
| Frequency     | HW-Device-Timelinkmicro-Tms6001-Frequency-SNMP-custom  | Check frequency status           |
| Gnss          | HW-Device-Timelinkmicro-Tms6001-Gnss-SNMP-custom       | Check GNSS receiver              |
| Satellites    | HW-Device-Timelinkmicro-Tms6001-Satellites-SNMP-custom | Check satellites                 |

> | Time          | HW-Device-Timelinkmicro-Tms6001-Time-SNMP-custom       | Check quality of time generation |

Reviewer: same as FR

More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).

### Collected metrics & status

Here is the list of services for this connector, detailing all metrics and statuses linked to each service.

<Tabs groupId="sync">
<TabItem value="Alarms" label="Alarms">

| Name               | Unit  |
|:-------------------|:------|
| alarms.total.count | count |

</TabItem>
<TabItem value="Antenna" label="Antenna">

| Name   | Unit  |
|:-------|:------|
| status | N/A   |

</TabItem>
<TabItem value="Frequency" label="Frequency">

| Name                               | Unit  |
|:-----------------------------------|:------|
| generation.frequency.quality.count | count |

</TabItem>
<TabItem value="Gnss" label="Gnss">

| Name   | Unit  |
|:-------|:------|
| status | N/A   |

</TabItem>
<TabItem value="Satellites" label="Satellites">

| Name                  | Unit  |
|:----------------------|:------|
| satellites.seen.count | count |

</TabItem>
<TabItem value="Time" label="Time">

| Name                          | Unit  |
|:------------------------------|:------|

> | generation.time.quality.count | count |

Reviewer: standard snmp prerequisites

</TabItem>
</Tabs>

3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector
is able to monitor a resource using a command like this one (replace the sample values by yours):

```bash
/usr/lib/centreon/plugins/centreon_timelinkmicro_tms6001_snmp.pl \
	--plugin=hardware::devices::timelinkmicro::tms6001::snmp::plugin \
	--mode=time \
	--hostname=10.0.0.1 \
	--snmp-version='2c' \
	--snmp-community='my-snmp-community'  \
	--warning-time-quality='' \
	--critical-time-quality=''
```

The expected command output is shown below:

```bash

> OK: quality of time generation: 85955 | 'generation.time.quality.count'=85955;;;;

Reviewer: same as FR
